tactics/ind_tables.mli

```diff
@@ -67,7 +67,7 @@ val pr_scheme_kind : 'a scheme_kind -> Pp.t
 
 val declare_definition_scheme :
   (internal : bool
-   -> univs:Entries.universes_entry
+   -> univs:(UState.named_universes_entry)
```

Comment:
Just curious, why is this in brackets?

Reply:
Remainder of a global `sed` substitution... Removed.